### PR TITLE
KLBP-305: Redirect to 401 page when patch breadcrumb fails.

### PR DESCRIPTION
**Pull request by andreir-suciu**

### CODE REVIEW

**Review by mariuspop86 — APPROVED** [review]

**Review by ioan-iacob-netea — APPROVED** [review]

**Review by iuliudascalu — APPROVED** [review]

File: src/hooks/useStep2Hook.ts [hunk 1]
```diff
@@ -172,10 +172,15 @@ const useStep2Hook = () => {
 
   const onClickPrevious = async (): Promise<void> => {
     dispatch(loadingActions.loading())
-    await patchBreadcrumbs(PREVIOUS_PAGE)
-    await getBreadcrumbs()
-    history.push(ROUTES.step0) // TODO this should be dynamic, for EI this should be enough
-    dispatch(loadingActions.loaded())
+    try {
```
Comment (mariuspop86) [inline src/hooks/useStep2Hook.ts:175]: Please add a test

**Review by mariapitechplus — APPROVED** [review]

File: src/hooks/useStep2Hook.ts (hunk 1 above)
Comment (andreir-suciu) [reply src/hooks/useStep2Hook.ts:175]: Done

File: src/interfaces/Breadcrumb.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ export interface IBreadcrumb {
   stepNumber: number
   titre: string
   visited?: boolean
+  revisit?: boolean
```
Comment (ioan-iacob-netea) [inline src/interfaces/Breadcrumb.ts:10]: Please sort the interface properties.

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

File: src/interfaces/Breadcrumb.ts (hunk 2 above)
Comment (andreir-suciu) [reply src/interfaces/Breadcrumb.ts]: Done.

**Review by mariuspop86 — APPROVED** [review]

**Review by ioan-iacob-netea — APPROVED** [review]

**Pull request merged**